charts/backingservices/values.yaml

```diff
@@ -29,43 +40,48 @@ srs:
     # To use your own Elasticsearch cluster, set srsStorage.provisionInternalESCluster to false and then
     # set the external Elasticsearch cluster URL and port details below when using an externally managed elasticsearch
     # Ensure that the specified endpoint is accessible from the kubernetes cluster pods.
+
     # The elasticsearch connection supports two authentication methods: basic authentication or AWS IAM role-based authentication.
-#    domain: managed-elasticsearch.acme.io
-#    port: 443
-#    protocol: https
-#     To configure basic authentication to your externally-managed Elasticsearch cluster, uncomment and add the
-#     parameter details: srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password
-#    basicAuthentication:
-#      username: "BASIC_AUTH_USERNAME"
-#      password: "BASIC_AUTH_PASSWORD"
-#     To configure AWS IAM role-based authentication to your externally-managed Elasticsearch cluster, uncomment
-#     and add the parameter details: srs.srsStorage.awsIAM and its associated region, srs.srsStorage.awsIAM.region
-#    awsIAM:
-#      region: "AWS_ELASTICSEARCH_REGION"
-
-#     To configure either authentication method, when the elasticsearch domain requires an open internet connection, uncomment to set this parameter to "true".
-#    requireInternetAccess: true
+    # domain: managed-elasticsearch.acme.io
+    # port: 443
+    # protocol: https
+    #     To configure basic authentication to your externally-managed Elasticsearch cluster, uncomment and add the
+    #     parameter details: srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password
+    # basicAuthentication:
+    #   username: "BASIC_AUTH_USERNAME"
+    #   password: "BASIC_AUTH_PASSWORD"
+    #     To configure AWS IAM role-based authentication to your externally-managed Elasticsearch cluster, uncomment
+    #     and add the parameter details: srs.srsStorage.awsIAM and its associated region, srs.srsStorage.awsIAM.region
+    # awsIAM:
+    #   region: "AWS_ELASTICSEARCH_REGION"
+    # To configure either authentication method, when the elasticsearch domain requires an open internet connection, uncomment to set this parameter to "true".
+    # requireInternetAccess: true
 
 # This section specifies the configuration for deploying an internal elasticsearch cluster for use with SRS.
 # The configuration for rest of the values defined under 'elasticsearch' are to define the elasticsearch cluster
 # based on helm charts defined at https://github.com/elastic/helm-charts/tree/master/elasticsearch and may be modified
 # as per runtime and storage requirements.
 elasticsearch:
   imageTag: 7.10.2
+
```

Comment:
Added a bunch of spacing just to make things look better

Reply:
Careful -- do not incur the wrath of the yaml linter.